test/index.js

```diff
@@ -132,6 +132,44 @@ describe('stand-in', function () {
     });
   });
 
+  describe('prototypes', function () {
+    it('replaces a prototype method', function (done) {
```

Comment:
It was actually the restoring that was failing. I moved my test and got rid of the duplicate one.